torch/utils/_pytree.py

```diff
@@ -741,8 +766,9 @@ def tree_map_only_(
     __type_or_types: TypeAny,
     func: FnAny[Any],
     tree: PyTree,
+    *rest: PyTree,
 ) -> PyTree:
-    return tree_map_(map_only(__type_or_types)(func), tree)
+    return tree_map_(map_only(__type_or_types)(func), tree, *rest)
```

Comment:
One concern is the `@map_only()` decorator only accepts a single-arg function. We need to expand that too.

Reply:
I can remove that change if it doesn't make sense, just did it for consistency